### DISCUSSION TITLE
UMD

### DISCUSSION THREAD

**Discussion by szkl** [family/connectkit#93] [Ideas]
Is it possible to have a [UMD](https://github.com/umdjs/umd) build of ConnectKit?

**Comment by 4x4notfound** [comment]
this would be great and speedup startup times in frameworks like Remix

**Comment by lochie** [comment]
Good idea! We've added this to our tasks but not sure how high of a priority this will be, though you're always welcome to set up a PR if you wish to set this up yourself 😊

**Reply by 4x4notfound** [reply]
In the meantime, for anyone visiting this ticket, here is how to can hack it wtith React 18+ https://github.com/kubelt/kubelt/blob/main/apps/passport/app/components/connect-button/ConnectButton.tsx
